basic_portfolio/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Basic Portfolio Page</title>
+<style>
+h1{
+	background-color: lavender;
+	height: 20%;
+	text-align: right;
+}
+#headertext li {
+	display: inline;
+	list-style: none;
```

Comment:
While this works, you have used the generic attribute for one particular trait.  Maybe use <a href="http://www.w3schools.com/cssref/pr_list-style-type.asp" target="_blank">list style type</a> instead of <a href="http://www.w3schools.com/cssref/pr_list-style.asp" target="_blank">generic list style<a> for this particular property